internal/models/sessions.go

```diff
@@ -69,13 +69,41 @@ type Session struct {
 	FactorID  *uuid.UUID `json:"factor_id" db:"factor_id"`
 	AMRClaims []AMRClaim `json:"amr,omitempty" has_many:"amr_claims"`
 	AAL       *string    `json:"aal" db:"aal"`
+
+	RefreshedAt *time.Time `json:"refreshed_at,omitempty" db:"refreshed_at"`
+	UserAgent   *string    `json:"user_agent,omitempty" db:"user_agent"`
+	IP          *string    `json:"ip,omitempty" db:"ip"`
```

Comment:
hmm do we want to validate that the IP passed in the request header matches the format specified by postgres' `inet`? or should we just let postgres deal with that?

Reply:
Good catch! I'll do that!

Reply:
Added this in GetIPAddress.